test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -72,6 +72,26 @@ def test_server_error
     end
   end
 
+  def test_http_request_from_record_with_referer_nil_or_absent
```

Comment:
Let's actually split this into two tests, `test_http_request_from_record_with_referer_nil` and `test_http_request_from_record_with_referer_absent`.